### PR TITLE
Test

### PR DESCRIPTION
**Pull request by macchiati**

### CODE REVIEW

File: unicodetools/org/unicode/tools/emoji/GenerateEmoji.java [hunk 1]
```diff
@@ -3490,9 +3490,10 @@ static void showCandidateStyle(CandidateStyle candidateStyle, String outFileName
 		+ "</ul>\n"
 		: "";
 	String top4 = candidateStyle != CandidateStyle.released
-		? "Where the last Sample cell is marked with the symbol "
-		+ NOT_NEW + " it indicates that a previously existing character is included, "
-		+ "and only because it has new gender and/or modifier variants.</p>\n"
+		? " Where the last Sample cell is marked with the symbol "
+		+ NOT_NEW + ", there is a row for an pre-existing character that has new gender and/or modifier variants. "
+		+ "For example, ➕ 5 Ⓒ ‧ 🏿 means that additional 5 skintone variants on the original character are added,  "
```
Comment (seanstewart-unicode) [inline unicodetools/org/unicode/tools/emoji/GenerateEmoji.java:3495]: Helpful addition here for those who need to make sense of the emoji counts keys!

File: unicodetools/org/unicode/tools/emoji/candidateData.txt [hunk 2]
```diff
@@ -218,7 +218,7 @@ U+1F636 U+200D U+1F32B U+FE0F
 Name=face in clouds
 Keywords=brain fog | confused | delirious | foggy | forgetful | head in clouds
 
-Proposal=L2/20-131
+Proposal=L2/19-391
```
Comment (seanstewart-unicode) [inline unicodetools/org/unicode/tools/emoji/candidateData.txt:221]: Verified correct document. These pesky IDs are hard to wrangle.

File: unicodetools/org/unicode/tools/emoji/candidateData.txt [hunk 3]
```diff
@@ -234,7 +234,7 @@ Name=woman with beard: medium-dark skin tone
 U+1F9D4 1F3FF U+200D U+2640
 Name=woman with beard: dark skin tone
 
-Proposal=L2/20-131
+Proposal=L2/19-391
```
Comment (seanstewart-unicode) [inline unicodetools/org/unicode/tools/emoji/candidateData.txt:237]: Verified correct document.

File: unicodetools/org/unicode/tools/emoji/candidateData.txt [hunk 4]
```diff
@@ -250,7 +250,7 @@ Name=man with beard: medium-dark skin tone
 U+1F9D4 1F3FF U+200D U+2642
 Name=man with beard: dark skin tone
 
-Proposal=L2/20-131
+Proposal=L2/19-377
```
Comment (seanstewart-unicode) [inline unicodetools/org/unicode/tools/emoji/candidateData.txt:253]: Verified correct document.

File: unicodetools/org/unicode/tools/emoji/candidateData.txt [hunk 5]
```diff
@@ -120,34 +120,63 @@ Proposal=L2/19-396, L2/19-400
...
+Keywords=dizzy | hypnotized | spiral | trouble | whoa
+
+Proposal=L2/19-380
+After=💔
+U+2764 U+FE0F U+200D U+1FA79
+Name=mending heart
+Keywords=healthier | improving | mending | recovering | recuperating | well
+
+Proposal=L2/20-128
+After=💔
```
Comment (seanstewart-unicode) [inline unicodetools/org/unicode/tools/emoji/candidateData.txt]: Shouldn't be an issue that it's the same `After=` as above.

Comment (nedley) [reply unicodetools/org/unicode/tools/emoji/candidateData.txt]: Ken Lunde confirmed [here](https://docs.google.com/document/d/1sfPeetALAwcytdLx6vc-HG5N2vsHhlhuklww8d_2S2g/edit?disco=AAAAGgWcgxI).

**Comment by markusicu** [comment]
Pending PR here, 11 months old, and called "test". Does this want to be renamed and merged, or re-reviewed, or just closed?

**Pull request closed without merging**